documentation/book/appendix_metrics.adoc

```diff
@@ -14,7 +14,7 @@ endif::InstallationAppendix[]
 
 === Kafka Metrics Configuration
 
-Strimzi uses the https://github.com/prometheus/jmx_exporter[Prometheus JMX Exporter] to export JMX metrics from Kafka and ZooKeeper to a Prometheus HTTP metrics endpoint that is scraped by Prometheus server.
+{ProductName} uses the https://github.com/prometheus/jmx_exporter[Prometheus JMX Exporter] to export JMX metrics from Kafka and ZooKeeper to a Prometheus HTTP metrics endpoint that is scraped by Prometheus server.
```

Comment:
I think this should use the proper link format as we use in other places (E.g. `link:https://kafka.apache.org/documentation/#basic_ops_racks[Kafka racks documentation^]`). It maybe should be also in attributed to make it easy to change in all places if needed.